### DISCUSSION TITLE
I get something different from everyone else

### DISCUSSION THREAD

**Discussion by Akaariii** [esperecyan/VRMConverterForVRChat#20] [Q&A] [answered]
When I download the VRM converter from BOOTH, I have a zip file, and when I open it I get a folder full of scripts and assets and I don't know what to do with it. I'm using a mac btw, can someone help me? :c
![Captura de Pantalla 2021-07-06 a la(s) 19 15 51](https://user-images.githubusercontent.com/87047366/124681653-9fa44f80-de8e-11eb-95e1-10782e541193.png)
![Captura de Pantalla 2021-07-06 a la(s) 19 15 43](https://user-images.githubusercontent.com/87047366/124681656-a0d57c80-de8e-11eb-80d3-79e8909533f0.png)

**Comment by esperecyan** [comment, marked as answer]
I did a search and found a workaround.
https://community.gamedev.tv/t/mac-users-solution-no-unitypackage-asset-in-the-zip-file-attached/97470
